Realm/RLMProperty.mm

```diff
@@ -349,8 +356,17 @@ - (instancetype)initSwiftPropertyWithName:(NSString *)name
 
     NSString *rawType;
     bool readOnly = false;
-    [self parseObjcProperty:property readOnly:&readOnly rawType:&rawType];
-    if (readOnly) {
+    bool isComputed = false;
+    [self parseObjcProperty:property readOnly:&readOnly computed:&isComputed rawType:&rawType];
+    if (!readOnly && isComputed) {
+        // Check for lazy property.
+        NSString *backingPropertyName = [NSString stringWithFormat:@"%@.storage", name];
+        if (class_getInstanceVariable([obj class], backingPropertyName.UTF8String)) {
```

Comment:
Don't quite follow why checking for lazy property occurs if the property is computed?
or do lazy properties resolve as being computed when parsing?
also seems strange since lazy vars are not supported in Realm Swift object classes

![image](https://user-images.githubusercontent.com/274318/34705780-14abeeb6-f557-11e7-9b1b-fb0ee4dc0a0d.png)

Reply:
https://github.com/realm/realm-cocoa/issues/2840#issuecomment-356177886